chef-server-stats/chef-server-stats

```diff
@@ -295,6 +316,7 @@ output.merge!(get_authz_stats(services))
 output.merge!(get_redis_stats(services))
 output.merge!(get_server_status)
 output.merge!(get_folsom_metrics(chef_server_12))
+output.merge!(get_pooler_metrics(chef_server_12))
```

Comment:
will this use the same keys? Is the idea here that if you have pooler_metrics, it wins, or are there separate keys that one needs to start watching? Also, will pooler_metrics always work, or does it require a specific version of chef server?